ChangeLog

```diff
@@ -1,3 +1,33 @@
+3.0.0
+=====
+- Bumped gRPC dependency to 2.29.0. Since this is a breaking change as per the gRPC team, we
+  have made a corresponding bump in the major version of this library. See
+  https://github.com/grpc/proposal/blob/master/L57-csharp-new-major-version.md to learn more about
+  the gRPC changes.
+- The library dependencies were bumped as listed below:
+  - Google.Protobuf: 3.12.2.
+  - Grpc.Core and related assemblies: 2.29.0
+  - Google.Api.Gax.Grpc and related assemblies: 3.0.0
+  - Google.LongRunning: 2.0.0
+  - .NET: netstandard2.0;net472 for libraries, and netcoreapp3.0;net472 for executables.
+- Fixed https://github.com/googleads/google-ads-dotnet/issues/192
+- Fixed https://github.com/googleads/google-ads-dotnet/issues/163
+- Fixed https://github.com/googleads/google-ads-dotnet/issues/120
+- Removed several method flattenings that involved validateOnly and partialFailure parameters. The
```

Comment:
I would list all the methods that are impacted here.

Reply:
That's pretty much every single mutate method in every single service, so it's a long list.